addons/hr_recruitment_survey/security/hr_recruitment_survey_security.xml

```diff
@@ -24,5 +144,46 @@
             <field name="perm_read" eval="1"/>
             <field name="perm_create" eval="0"/>
         </record>
+        <record id="survey_survey_recruitment_interviewer" model="ir.rule">
+            <field name="name">Survey: recruitment interviewer: send surveys to applicants for which they are set as interviewer</field>
+            <field name="model_id" ref="survey.model_survey_survey"/>
+            <field name="domain_force">[
+                '|', ('job_ids.interviewer_ids', 'in', user.id),
+                     ('job_ids.application_ids.interviewer_ids', 'in', user.id)
+                ]</field>
+            <field name="groups" eval="[(4, ref('hr_recruitment.group_hr_recruitment_interviewer'))]"/>
+            <field name="perm_unlink" eval="0"/>
+            <field name="perm_write" eval="0"/>
+            <field name="perm_read" eval="1"/>
+            <field name="perm_create" eval="0"/>
+        </record>
+
+        <record id="survey_question_recruitment_interviewer" model="ir.rule">
+            <field name="name">Survey: recruitment interviewer: send surveys to applicants for which they are set as interviewer</field>
+            <field name="model_id" ref="survey.model_survey_question"/>
+            <field name="domain_force">[
+                '|', ('survey_id.job_ids.interviewer_ids', 'in', user.id),
+                     ('survey_id.job_ids.application_ids.interviewer_ids', 'in', user.id)
+                ]</field>
+            <field name="groups" eval="[(4, ref('hr_recruitment.group_hr_recruitment_interviewer'))]"/>
+            <field name="perm_unlink" eval="0"/>
+            <field name="perm_write" eval="0"/>
+            <field name="perm_read" eval="1"/>
+            <field name="perm_create" eval="0"/>
+        </record>
+
+        <record id="survey_invite_recruitment_interviewer" model="ir.rule">
+            <field name="name">Survey invite: recruitment interviewer: send surveys to applicants for which they are set as interviewer</field>
+            <field name="model_id" ref="survey.model_survey_invite"/>
+            <field name="domain_force">[
```

Comment:
Would also probably add survey_type being recruitment to be explicit and clear about that (in which case, valid for all rules)